### PR TITLE
Opening Reception at DjangoCon US and board game night

### PR DESCRIPTION
**Pull request by drewbrew**
Changes proposed in this PR:

- Adds a blog post about the opening reception for Carol to tweet about.

### CODE REVIEW

File: _posts/2022-10-13-opening-reception.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+author: Lacey Williams Henschel
```
Comment (logankilpatrick) [inline _posts/2022-10-13-opening-reception.md]:
```suggestion
author: DjangoCon US Organizers
```

Comment (logankilpatrick) [reply _posts/2022-10-13-opening-reception.md]: I am not against giving Lacey credit but we shouldn't publish this without getting her approval first

File: _posts/2022-10-13-opening-reception.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+**When**: 5:45-7:00 pm, Monday, September 23
+
+**Where**: West Lawn at the Marriott (on the other side of the pool from the Marriott restaurant)
+
+**What**: Join us for some food and non-alcoholic drinks to relax after the first day of talks! We'll have light appetizers for you to enjoy.
+
+Also don't forget our other social event, board game night!
+
+## Board Game Night
```
Comment (logankilpatrick) [inline _posts/2022-10-13-opening-reception.md:22]: Don't we already have this in the other post?

Comment (drewbrew) [reply _posts/2022-10-13-opening-reception.md:22]: Yep, I was rehashing it to have an all in one page for Carol to tweet about.

**Pull request merged**